src/svindexer.ts

```diff
@@ -117,6 +118,27 @@ export class SystemVerilogIndexer {
         this._userDefines = defines.map(s => s.split("=", 2)).map(s => [s[0], s[1], s[1] == undefined ? [] : SystemVerilogPreprocessor.tokenize(s[1])]);
     }
 
+    setLibraries(libraries: string[], excludes: string[]) {
+        let _indexGlob = (func) => {
+            const pattern = libraries.length == 1 ? libraries[0] : '{' + libraries.join(",") + '}';
+            glob(pattern, {cwd: this._rootPath, ignore: excludes, follow: true, realpath: true}, func);
+        };
+        _indexGlob((err, files) => {
+            if (err) {
+                ConnectionLogger.error(err);
+            }
+            else if (files.length > 0) {
+                if (!genutils_1.isStringListEqual(files, this._libFiles)) {
```

Comment:
Please remove the genutils_1 part. Just isStringListEqual should be fine.

Reply:
Fixed.